.github/workflows/release.yaml

```diff
@@ -25,39 +27,64 @@ jobs:
           poetry config virtualenvs.create false
           poetry install -E all
 
+      - name: Dump last commit message to file
```

Comment:
What if the PR has multiple commits? We don't care about them?

FYI, it seems like we would our own semantic release process here.

Reply:
That is not possible, we require squash commit from PRs and the PR description becomes the commit message:
![image](https://github.com/user-attachments/assets/eab16b4f-1774-4662-8eda-fa1e5d15ef1e)